### DISCUSSION TITLE
xplat.browser template in the net9 support

### DISCUSSION THREAD

**Discussion by gritsenko** [AvaloniaUI/Avalonia.Templates#271] [General]
Hey, @maxkatz6 ! It looks like in the net9 there is a new approach of deploying static files. I mean using wwwroot instead of AppBundle and     
```
<WasmExtraFilesToDeploy Include="AppBundle/**" />
```

https://github.com/dotnet/runtime/issues/77285#issuecomment-1774847294

**Comment by gritsenko** [comment]
I was able to run my app, after changed AppBundle directory name to wwwroot, added there avalonia.js, sw.js and storage.js into it (from previous net8 build) and changed their property "Copy" to the "Copy if newer" value

**Comment by maxkatz6** [comment]
Correct, we are using wwwroot since 11.1 now: https://github.com/AvaloniaUI/avalonia-dotnet-templates/tree/43c5d489827688e412c98722ba1963039c66e9a5/templates/csharp/xplat/AvaloniaTest.Browser
